test/multiverse/lib/multiverse/suite.rb

```diff
@@ -287,7 +285,7 @@ def generate_gemfile(gemfile_text, env_index, local = true)
         f.puts "gem 'minitest-stub-const', '~> 0.6', require: false"
 
         # pin webrick until we investigate why 1.8.1 breaks things
-        f.puts "gem 'webrick', '< 1.8.0'" if RUBY_VERSION > '2.3.0'
+        f.puts "gem 'webrick', '< 1.8.0'" if RUBY_VERSION > '2.4.0'
```

Comment:
we should be able to delete the if from this since it was only making sure this didn't happen on 2.2 🎉 
```suggestion
        f.puts "gem 'webrick', '< 1.8.0'"
```

Reply:
Makes sense :) [Done](https://github.com/newrelic/newrelic-ruby-agent/pull/1778/commits/10d7ca76aac24e3ea2a566fc48164d674342eb70#diff-5c92585b1f81da4bb4c96bff0e84e668725467ce36ed0b1915c68e866c50bbfdR288).